### ISSUE TITLE
Move Acknowledgements out of CocoaPods

### ISSUE THREAD

**Issue by kylef** [CocoaPods/CocoaPods#2465]
The work of this has been started in two repositories:
- https://github.com/CocoaPods/cocoapods-install-metadata
- https://github.com/CocoaPods/CPDAcknowledgements

With CocoaPods 0.34.0 shipping with plugin support, this unblocks this feature and will allow its completion.

**Comment by segiddins** [comment]
@kylef what's the status on this?

**Comment by kylef** [comment]
I think @orta has made some progress.

**Comment by orta** [comment]
This mostly works,

**Comment by Ashton-W** [comment]
This is not addressing #1924, which was closed in preference to this issue.

**Comment by jcayzac** [comment]
Maybe having an optional `spec.license[:private]` bool attribute would allow for filtering out specs that set it?

**Comment by levigroker** [comment]
+1 This would be quite nice to have. Thanks!

**Comment by segiddins** [comment]
@orta can we close this as being resolved by `cococapods-acknowledgements`?
